Cannot release due to missing Prisma dependency

Thanks for releasing this code, Cloudflare is my frontend stack for years now, it's great to have this project to not all invent the wheel ourselves.

I tried releasing after a fresh `npx degit redwoodjs/sdk/starters/standard <project-name>` as stated in the quick start.

What I exactly did is:
1 - degit the standard starter
2 - npm i
3 - npm run dev (worked on 2nd run)
4 - npm run generate
5 - npm release

I then get this error:
```
Uncaught Error: No such module ".prisma/client/default".
    imported from "worker.js"
   [code: 10021]
```

I googled around but don't know where it's coming from. I run node v20.15.0

Full logs:
```

> @redwoodjs/starter-standard@1.0.0 release
> rw-scripts ensure-deploy-env && npm run clean && prisma generate && RWSDK_DEPLOY=1 npm run build && wrangler deploy

Do you want to proceed with deployment? (y/N): y
Ensuring deployment environment is ready...
Checking Cloudflare account setup...
Ensuring worker innergrowth exists...
Found env.DB usage, checking D1 database setup...
D1 database already configured in wrangler.jsonc, skipping creation
Found auth usage, checking secret setup...
AUTH_SECRET_KEY secret already exists in Cloudflare, skipping
Found WEBAUTHN usage, checking WebAuthn setup...
WEBAUTHN_APP_NAME already set to "innergrowth" in wrangler.jsonc
Checking migration status...
Migrations are up to date.

Deployment initialization complete!

> @redwoodjs/starter-standard@1.0.0 clean
> npm run clean:vite


> @redwoodjs/starter-standard@1.0.0 clean:vite
> rm -rf ./node_modules/.vite

Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma

✔ Generated Prisma Client (v6.5.0) to ./node_modules/.prisma/client in 66ms

Start by importing your Prisma Client (See: https://pris.ly/d/importing-client)

Tip: Curious about the SQL queries Prisma ORM generates? Optimize helps you enhance your visibility: https://pris.ly/tip-2-optimize


> @redwoodjs/starter-standard@1.0.0 build
> vite build

vite v6.3.4 building for production...
✓ 95 modules transformed.
dist/client/.vite/manifest.json                              1.19 kB │ gzip:  0.32 kB
dist/client/assets/_virtual_use-client-lookup-CU9v9mL_.js    0.37 kB │ gzip:  0.27 kB │ map:   0.14 kB
dist/client/assets/client-Sy-M3uo7.js                        0.48 kB │ gzip:  0.32 kB │ map:   1.52 kB
dist/client/assets/Login-B1LpQLoc.js                         9.89 kB │ gzip:  3.34 kB │ map:  40.91 kB
dist/client/assets/client-D2nE7LdD.js                       53.72 kB │ gzip: 17.36 kB │ map: 189.76 kB
dist/client/assets/client-CrFvO51n.js                      174.88 kB │ gzip: 55.29 kB │ map: 835.98 kB
✓ built in 634ms
vite v6.3.4 building SSR bundle for production...
".prisma/client/default" is imported by ".prisma/client/default?commonjs-external", but could not be resolved – treating it as an external dependency.
✓ 302 modules transformed.
dist/worker/.dev.vars        0.17 kB
dist/worker/wrangler.json    1.33 kB
dist/worker/worker.js      488.30 kB │ map: 1,944.37 kB
✓ built in 1.31s

 ⛅️ wrangler 4.14.1
-------------------

Using redirected Wrangler configuration.
 - Configuration being used: "dist/worker/wrangler.json"
 - Original user's configuration: "wrangler.jsonc"
 - Deploy configuration file: ".wrangler/deploy/config.json"
🌀 Building list of assets...
✨ Read 13 files from the assets directory /Users/sjoerd/Werk/liefde.ai/innergrowth/dist/client
🌀 Starting asset upload...
No updated asset files to upload. Proceeding with deployment...
Total Upload: 476.85 KiB / gzip: 143.42 KiB
Your Worker has access to the following bindings:
- Durable Objects:
  - SESSION_DURABLE_OBJECT: SessionDurableObject
- D1 Databases:
  - DB: innergrowth-pleased-eagle (08ede69c-e088-4b26-8da2-f997a10fa676)
- Assets:
  - Binding: ASSETS
- Vars:
  - WEBAUTHN_APP_NAME: "innergrowth"

✘ [ERROR] A request to the Cloudflare API (/accounts/dc35da3dbc4f5c53c212d5ae9b95f291/workers/scripts/innergrowth) failed.

  Uncaught Error: No such module ".prisma/client/default".
    imported from "worker.js"
   [code: 10021]

  
  If you think this is a bug, please open an issue at:
  https://github.com/cloudflare/workers-sdk/issues/new/choose

```

--- a/sdk/src/vite/checkIsUsingPrisma.mts
+++ b/sdk/src/vite/checkIsUsingPrisma.mts
@@ -1,0 +1,7 @@
+import enhancedResolve from "enhanced-resolve";
+
+export const checkIsUsingPrisma = ({
+  projectRootDir,
+}: {
+  projectRootDir: string;
+}) => Boolean(enhancedResolve.sync(projectRootDir, "@prisma/client"));

--- a/sdk/src/vite/checkIsUsingPrisma.mts
+++ b/sdk/src/vite/checkIsUsingPrisma.mts
@@ -4,4 +4,10 @@ export const checkIsUsingPrisma = ({
   projectRootDir,
 }: {
   projectRootDir: string;
-}) => Boolean(enhancedResolve.sync(projectRootDir, "@prisma/client"));
+}) => {
+  try {
+    return Boolean(enhancedResolve.sync(projectRootDir, "@prisma/client"));
+  } catch {
+    return false;
+  }
+};

--- a/sdk/src/vite/redwoodPlugin.mts
+++ b/sdk/src/vite/redwoodPlugin.mts
@@ -19,6 +19,7 @@ import { findWranglerConfig } from "../lib/findWranglerConfig.mjs";
 import { pathExists } from "fs-extra";
 import { injectVitePreamble } from "./injectVitePreamblePlugin.mjs";
 import { vitePreamblePlugin } from "./vitePreamblePlugin.mjs";
+import { checkIsUsingPrisma } from "./checkIsUsingPrisma.mjs";
 
 export type RedwoodPluginOptions = {
   silent?: boolean;
@@ -64,8 +65,7 @@ export const redwoodPlugin = async (
     })`npm run dev:init`;
   }
 
-  const usesPrisma = await $({ reject: false })`pnpm prisma --version`;
-  const isUsingPrisma = usesPrisma.exitCode === 0;
+  const isUsingPrisma = checkIsUsingPrisma({ projectRootDir });
 
   // context(justinvdm, 10 Mar 2025): We need to use vite optimizeDeps for all deps to work with @cloudflare/vite-plugin.
   // Thing is, @prisma/client has generated code. So users end up with a stale @prisma/client

